tools/clang/test/CodeGenSPIRV/sm6_6.descriptorheap.acbuffer.hlsl

```diff
@@ -9,6 +9,7 @@
 // CHECK:   OpDecorate [[resource_heap_abuffer]] Binding 0
 // CHECK:   OpDecorate [[resource_heap_abuffer_counter:%[_a-zA-Z0-9]+]] DescriptorSet 0
 // CHECK:   OpDecorate [[resource_heap_abuffer_counter]] Binding 1
+// CHECK-NOT: OpDecorate %_runtimearr_type_ACSBuffer_counter ArrayStride
```

Comment:
This should also check the BufferBlock decoration presence no? (Same in the second test)

Reply:
done